test/public/lhcFills/overview.test.js

```diff
@@ -264,7 +265,27 @@ module.exports = () => {
         await expectInnerText(page, efficiencyExpect.selector, efficiencyExpect.value);
     });
 
-    it('should successfully toggle to stable beam only', async () => {
+    it('should successfully display filter elements', async () => {
+        const filterSBExpect = { selector: '.w-30', value: 'Stable Beams Only' };
```

Comment:
The selector is definitely too general and will create problems for sure in the future.
Could you please make use of an `id` instead?